meetings/agenda_Mtg2_2-6-2020.md

```diff
@@ -0,0 +1,130 @@
+February 6<sup>th</sup>, 2020
+
+# NumPy + SMT Meeting 2
+
+**<span class="underline">Agenda</span>**
```

Comment:
Comment for next times (no neeed to change now): the explicit html/css like things are best left out, it's mostly spam in Markdown docs. Just using nested headers is usually fine, and also i believe you can get a horizontal line with `-----` as a standalone line

Reply:
Yup - these are artifacts from `pandoc`. We were originally going to try to use GitHub to collaborate with the survey team, but we have since landed on google docs. I was originally trying to `pandoc` between `docx` and `md` to keep consistent copies both here and with the survey team for archival purposes, but this is too cumbersome. Moving forward I think I'm just going to reorganize the `meetings` to link directly to the google docs :)